packages/runtime-dom/src/modules/props.ts

```diff
@@ -28,7 +28,10 @@ export function patchDOMProp(
     // store value as _value as well since
     // non-string values will be stringified.
     el._value = value
-    el.value = value == null ? '' : value
+    const newValue = value == null ? '' : value
+    if (el.value.length !== newValue.length || el.value !== newValue) {
```

Comment:
Under what circumstance can two values be strictly equal but have different `.length`? 🤔

Reply:
I used to compare two values directly, but considering that sometimes the value is a very long string, comparing the length first may help improve performance. Although I believe that the engine has done enough optimization for string comparison, the performance test results show that comparing length first and if lengths are equal then comparing content has a weak performance advantage.

Reply:
I've test again. it's my fault. although it can improve the performance in the best case, it will be about 5% slower in the normal scenario.